components/Common/CodeBox/index.stories.tsx

```diff
@@ -0,0 +1,130 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import { MDXRemote } from 'next-mdx-remote';
+import { serialize } from 'next-mdx-remote/serialize';
+import rehypePrettyCode from 'rehype-pretty-code';
+import type { HighlighterOptions } from 'shiki';
+import { getHighlighter, setCDN } from 'shiki';
+import shikiNordTheme from 'shiki/themes/nord.json';
+
+import CodeBox from '@/components/Common/CodeBox/index';
+import codeTabsResolver from '@/mdx/plugins/codeTabsResolver.mjs';
+import { mdxComponents } from '@/providers/mdxProvider';
+import { SUPPORTED_LANGUAGES } from '@/shiki.config.mjs';
+
+setCDN('https://unpkg.com/shiki');
+
+type Story = StoryObj<typeof CodeBox>;
+type Meta = MetaObj<typeof CodeBox>;
+
+export const SingleFile: Story = {
+  args: {
+    children: `\`\`\`javascript showLineNumbers
+const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World');
+});
+
+server.listen(port, hostname, () => {
+  console.log(\`Server running at http://\${hostname}:\${port}/\`);
+});
+\`\`\``,
+  },
+};
+
+export const MultipleFiles: Story = {
+  args: {
+    children: `\`\`\`mjs|cjs showLineNumbers labels="MJS,CJS"
```

Comment:
So showLineNumbers is something from Shiki? Is this labels field also?

Reply:
`showLineNumbers` is handled by `rehype-pretty-code`. It just adds an attribute `data-line-numbers`, but since we're always showing them I've just removed it.

`labels` is a custom meta property resolved by the `codeTabsResolver`, so we can display custom tab titles.

Reply:
Right but lets please remove all of this.